1forge/README.md

```diff
@@ -27,6 +27,8 @@
 |    ✅     | `base`, `from` |      The symbol of the currency to query      | [List](https://1forge.com/currency-list) |             |
 |    ✅     | `quote`, `to`  |   The symbol of the currency to convert to    | [List](https://1forge.com/currency-list) |             |
 |    🟡     |   `quantity`   | An additional amount of the original currency |                                          |             |
+|    🟡     |   `overrides`   | If base provided is found in overrides, that will be used  | [Format](../presetSymbols.json)|             |
```

Comment:
Maybe we should have some root level documentation on shared `input parameters` like we do for `meta`. Don't remove these - but something to think about to make these docs "scale".